packages/compiler/src/app/builders/ng-app.builder.ts

```diff
@@ -343,10 +343,18 @@ export class NgAppBuilder {
     }
 
     // | styles.css
+    // if (diff.globalStyleChanged || _.has(diff, 'prev.globalStyle')) {
```

Comment:
?

Reply:
This is the code where it finds the style code (both css and scss) in the user app and dumps it into a (compiled) css file. I changed it so that it dumps the code into a (compiled) scss file. I removed the commented code.